ui/models.py

```diff
@@ -42,45 +44,104 @@ class Video(models.Model):
     description = models.TextField(blank=True)
     source_url = models.URLField()
     moira_lists = models.ManyToManyField(MoiraList)
+    status = models.TextField(max_length=24, blank=True)
+    s3_subkey = models.UUIDField(unique=True, null=False, blank=False, default=uuid4)
+
+    @property
+    def s3_key(self):
+        """
+        Avoid duplicate S3 keys/filenames when transferring videos rom Dropbox
```

Comment:
ok... I asked because maybe it is not the best to have it as a property: it looks more like a function to generate an unique key.
Being a property it is more like the key of an object that already exists (not sure I am clear).
Anyway feel free to leave as is if you think it is clear enough